rbac/api/cross_access/model.py

```diff
@@ -51,7 +51,8 @@ def validate_input_value(self):
         if self.status not in STATUS_LIST:
             raise ValidationError(f'Unknown status "{self.status}" specified, {STATUS_LIST} are valid inputs.')
 
-        [self.validate_date(date) for date in [self.start_date, self.end_date]]
+        if self.status != "expired":
```

Comment:
Overall LGTM. Maybe I missed sth, but if the CAR status is "pending" and gets updated, it would skip this date validation here, which seems not desired

Reply:
It's been a minute, but if I remember right I had to put this in because otherwise, when we set a requests's status to `expired`, we'd get complaints that the dates are past. Those dates being past is expected, though. since the request is expiring. With this check, any request that's not already in the `expired` state should still get the "in the past" check.

-

Reply:
Ah, I was out of my mind, I thought it was == "expired"